<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetCheckFileInfo</class>
 <widget class="QDialog" name="GetCheckFileInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>82</y>
     <width>249</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>检查单位：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditCompany"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>检 察 员：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditCheckMan"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
